projects/project_tests.py

```diff
@@ -119,3 +119,71 @@ def test__projects__admin__ProjectResource__1(browser, project_dict):
 def test__projects__models__1(db, project):
     """Project gets represented by its title."""
     assert str(project) == project.title
+
+
+def test__projects__admin__ProjectAdmin__1(browser, project_dict):
+    """Export the projects date to ics."""
+    project_dict['begind_date'] = datetime(
+        year=2022,
+        month=9,
+        day=26,
+        tzinfo=TZ,
+    )
+    project_dict['title'] = 'new_project'
+    proj1 = create_project(project_dict)
+
+    project_dict['begin_date'] = datetime(
+        year=2021,
+        month=9,
+        day=26,
+        tzinfo=TZ,
+    )
+    project_dict['title'] = 'old_project'
+    proj2 = create_project(project_dict)
+
+    browser.login_admin()
+    browser.open(A_PROJ_URL)
+    browser.follow('Export dates')
+
+    assert browser.headers['Content-Type'] == 'text/calendar'
+    assert proj1.title in str(browser.contents)
+    assert proj2.title in str(browser.contents)
+
+
+def test__projects__admin__ProjectAdmin__2(browser, project_dict):
+    """Export the projects date of this year to ics."""
+    project_dict['begind_date'] = datetime(
+        year=2022,
```

Comment:
This test will probably fail in January 2023. There is a way to mock out time with [freezegun](https://pypi.org/project/pytest-freezegun/), for integration see for example https://github.com/gocept/md-diga-portal/blob/d7c25b6282f980299871ae748b78f181cec6d851/src/md.digabackend/src/md/digabackend/middleware_tests.py with the use of the `freezer` fixture. So we could move to a time in the future and assert, that other projects are returned.

Reply:
In previous tests I just used `datetime.now().year` for this. Because I don't thing a date in the future is needed here I would do this the same way here [9d3da0a](https://github.com/gocept/ok_tools/pull/88/commits/9d3da0a5e30b387a73912dc0df4af3941ba92258)